app/src/main/res/values/colors.xml

```diff
@@ -8,4 +8,5 @@
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
     <color name="colorDoveGray">#686868</color>
+    <color name="Monza">#E50914</color>
```

Comment:
sai nè. đặt lại tên các colors khác nữa cho chuẩn 
https://chir.ag/projects/name-that-color/#6195ED

Reply:
vâng anh